scriptmodules/emulators/ppsspp.sh

```diff
@@ -25,9 +26,7 @@ function depends_ppsspp() {
 }
 
 function sources_ppsspp() {
-    local branch="$1"
-    [[ -z "$branch" ]] && branch="master"
-    gitPullOrClone "$md_build/ppsspp" https://github.com/hrydgard/ppsspp.git "$branch"
+    gitPullOrClone
     cd "ppsspp"
```

Comment:
Is the `cd` still needed if cloning goes directly to `$md_build` ?

Reply:
Actually this needs to be put back to ppsspp subfolder.